src/main/java/io/openbpm/control/service/job/impl/JobServiceImpl.java

```diff
@@ -171,4 +199,27 @@ protected List<JobQueryDtoSortingInner> createSortOptions(Sort sort) {
 
         return jobQueryDtoSortingInners;
     }
+
+    protected String fallbackGetHistoryStacktrace(String jobId) {
```

Comment:
Implement a custom query in the EngineRestClient class, see examples of queries in the class.
https://github.com/openbpm-platform/openbpm-control/blob/main/src/main/java/io/openbpm/control/service/client/EngineRestClient.java